CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ OPTION(ENABLE_FREETYPE "Enable Freetype2 support" 0)
 OPTION(ENABLE_FONTCONFIG "Enable FontConfig support" 0)
 OPTION(ENABLE_WEBP "Enable WebP support" 0)
 OPTION(ENABLE_RAQM "Enable RAQM support" 0)
+OPTION(ENABLE_ALL_OPTIONS "Enable all the options described above" 0)
```

Comment:
I think few users will use this option, which means they have to install all the dependencies. We prefer users can enable options one by one, which make them to check the options carefully.